csharp_dotnetcore/Using-Cards/Using_Cards/readme.md

```diff
@@ -0,0 +1,27 @@
+﻿This sample demostrates the use of rich content using cards.
+# Concepts introduced in this sample
+## Rich Cards
+Most channels support rich content.  In this sample we explore the different type od rich cards your bot can use.
+## To try this sample
+- Clone the repository.
+```bash
+git clone https://github.com/microsoft/botbuilder-samples.git
+```
+ ### Visual studio
+- Navigate to the samples folder (`BotBuilder-Samples\csharp_dotnetcore\6.Using_Cards`) and open 6.Using_Cards.sln in Visual Studio 
```

Comment:
recomment we NOT have a .sln file. Have people open the .csproj file.

Reply:
removed